eventing/samples/cronjob-source/channel.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Channel
+metadata:
+  name: cronjob-test
+  namespace: default
```

Comment:
I recommend omitting the namespace so that it can be picked up from the current `kubectl` context.